src/components/photography/article-list.js

```diff
@@ -136,14 +140,14 @@ const List = styled.ul`
   }
 
   ${ItemTitle}, ${ItemExcerpt}, ${Date} {
-    color: #F7F7F7;
+    color: ${colorGrayscale.gray100};
   }
   ${Item}:hover {
     ${ItemTitle}, ${Date} {
       color: black;
```

Comment:
Do we need to change `black` to `colorGrayscale.black`

Reply:
fixed ([d30b42](https://github.com/twreporter/twreporter-react/pull/2772/commits/d30b42cd3cca96f616a7bdaf6e20d94e6c523100))